.github/workflows/supported_storages_compatibility_check.yaml

```diff
@@ -0,0 +1,728 @@
+          HEAP_NEWSIZE: 512m
+        ports:
+          - 9042:9042
+
+    steps:
+      - uses: actions/checkout@v3
+
+      - name: Set up JDK 8
+        uses: actions/setup-java@v3
+        with:
+          java-version: '8'
+          distribution: 'temurin'
+
+      - name: Setup and execute Gradle 'integrationTestCassandra' task
+        uses: gradle/gradle-build-action@v2
+        with:
+          arguments: integrationTestCassandra
+
+      - name: Upload Gradle test reports
+        if: always()
+        uses: actions/upload-artifact@v3
+        with:
+          name: cassandra_3.0_integration_test_reports
+          path: core/build/reports/tests/integrationTestCassandra
+
+  cassandra-3-11-integration-test:
+    runs-on: ubuntu-latest
+
+    services:
+      cassandra:
+        image: cassandra:3.11.15
+        env:
+          MAX_HEAP_SIZE: 2048m
+          HEAP_NEWSIZE: 512m
+        ports:
+          - 9042:9042
+
+    steps:
+      - uses: actions/checkout@v3
+
+      - name: Set up JDK 8
+        uses: actions/setup-java@v3
+        with:
+          java-version: '8'
+          distribution: 'temurin'
+
+      - name: Setup and execute Gradle 'integrationTestCassandra' task
+        uses: gradle/gradle-build-action@v2
+        with:
+          arguments: integrationTestCassandra
+
+      - name: Upload Gradle test reports
+        if: always()
+        uses: actions/upload-artifact@v3
+        with:
+          name: cassandra_3.11_integration_test_reports
+          path: core/build/reports/tests/integrationTestCassandra
+
+  cosmos-integration-test:
+    runs-on: self-hosted
+
+    steps:
+      - uses: actions/checkout@v3
+
+      - name: Set up JDK 8
+        uses: actions/setup-java@v3
+        with:
+          java-version: '8'
+          distribution: 'temurin'
+
+      - name: Generate unique prefix using the epoch
+        run: |
+          echo "db_prefix=$(date +%s%3N)" >> $GITHUB_ENV
+
+      - name: Setup and execute Gradle 'integrationTestCosmos' task
+        uses: gradle/gradle-build-action@v2
+        with:
+          arguments: integrationTestCosmos -Dscalardb.cosmos.uri=${{ secrets.COSMOS_URI }} -Dscalardb.cosmos.password=${{ secrets.COSMOS_PASSWORD }} -Dscalardb.cosmos.database_prefix=${{ env.db_prefix }}_
+
+      - name : Delete gradle daemon log files
+        if: always()
+        #       Delete all files modified more than 3 days ago with the ".out.log" file extension located in the "/home/azureuser/.gradle/daemon"
+        #       folder hierarchy. These files accumulate over time and can end up using a lot of disk space
+        run : find /home/azureuser/.gradle/daemon -name "*.out.log" -type f -mtime +3 -exec rm -vf {} +
+
+      - name: Upload Gradle test reports
+        if: always()
+        uses: actions/upload-artifact@v3
+        with:
+          name: cosmos_integration_test_reports
+          path: core/build/reports/tests/integrationTestCosmos
+
+  dynamo-integration-test:
```

Comment:
Ditto. But I think we should upgrade the docker image version.

Reply:
I kept this job as we are testing with the latest image version.